src/internal/components/selectable-item/styles.scss

```diff
@@ -72,13 +73,20 @@
     }
   }
 
+  &.refresh {
+    &.selected {
+      border-color: awsui.$color-border-dropdown-item-selected;
+    }
+  }
+
```

Comment:
Why is this refresh class needed? Is there a reason this isn't just a design token?